ampersand-form-view.js

```diff
@@ -7,6 +7,7 @@ var result = require('amp-result');
 
 function FormView(opts) {
     opts = opts || {};
+    opts.autoRender = opts.autoRender !== undefined ? opts.autoRender : true;
```

Comment:
if `opts.autoRender` is defined and `false`, then the proposed statement would change it to `true` :$